### ISSUE TITLE
Possible to display paypal checkout after bottom profile

### ISSUE THREAD

**Issue by themak1985** [eileenmcnaughton/nz.co.fuzion.omnipaymultiprocessor#131]
So - I really like PayPal Checkout - we just started testing it - but we have a small issue.
We collect optional demographics information - once someone clicks paypal checkout - and clicks the paypal button - it immediately goes to the next page - which would typically be fine - but that means they skip the bottom profile section.  Any way to move things around?

Example: https://tarabnyc.org/donate/

**Comment by eileenmcnaughton** [comment]
So I guess it doesn't have to auto-submit - it seemed to make sense to do that but we could just look at them still having to click 'confirm'

**Comment by themak1985** [comment]
I think it does make sense to auto submit.

I am wondering if we are the more rare use case - having a bottom profile.  If it does not auto submit - it should say something - because its blank. 

I take it the placement is in core right?

**Comment by eileenmcnaughton** [comment]
Yes the placement is in core - I guess it can be altered by hooks / tpl editing but obviously you want a maintainable approach not something hacky (roll on flexible formbuilder)
